demos/redis/run_occlum_redis.sh

```diff
@@ -18,4 +18,7 @@ copy_bom -f $bomfile --root image --include-dir /opt/occlum/etc/template
 
 occlum build
 # 3. Run redis server
-occlum run /bin/redis-server /etc/redis.conf &
+# Default IPv6:
+occlum run /bin/redis-server --save "" --appendonly no &
```

Comment:
Maybe comment the ipv4 in redis.conf is better than change the execution command.

Reply:
Done.